web/modules/custom/genlyd_maps/js/genlyd_maps_tabs.js

```diff
@@ -19,7 +19,7 @@
         event.stopPropagation();
 
         if (!mapTab.is(':visible')) {
-          btn.text(Drupal.t('Show table'));
+          btn.text(Drupal.t('Show list'));
```

Comment:
As long as the javascript does not support Drupal.t, we should avoid using it at all.

For now my suggestion is using the danish words directly. If we, at a later time need to translate it, we should extend it so it works with Drupal.